README.md

```diff
@@ -177,13 +177,18 @@ For query such as:
 
 ```php
 $query = $model->where('count', 10)->limit(2);
-$last = $query->all()->last();
+$items = $query->all();
+$last = $items->last();
 ```
 
 Take the last item of this query result as the next "offset":
 
 ```php
 $nextPage = $query->after($last)->limit(2)->all();
+// or
+$nextPage = $query->afterKey($items->lastKey())->limit(2)->all();
+// or (for query without index condition only)
+$nextPage = $query->afterKey($last->getKeys())->limit(2)->all();
```

Comment:
hmm actually this isn't intuitive. The query should already know about index condition.

Reply:
how do we include that information and pass the keys from one request to another (ie; paginating from request to request).  The pagination query from page 1 to page 2 should be the same, but the last evaluated key/s is the only thing missing?

Reply:
Yeah, I know. It's tricky when the condition contains the index. I was wondering if documenting multiple ways would make things more confusing. 

I think I don't have any more changes. Do you want to take a look?